src/shared/components/case-list-filters/case-list-filters.component.spec.ts

```diff
@@ -1,1048 +0,0 @@
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
For completeness I think we do need unit tests checking emitting apply and reset notifications and rendering of wrapped component once data is loaded?

Reply:
Just added tests